isos/base/utils.sh

```diff
@@ -179,6 +179,10 @@ END {
 ensure_apt_packages() {
     local install
 
+    if [ -x /usr/local/bin/xorriso -a -x /usr/bin/cpio ]; then
```

Comment:
Why? We're passing in the package names we depend on to confirm and instead you've hardcoded a check for specific binaries.
I also see absolutely no correlation between this change and #7775

Reply:
this is just to trigger CI, nothing else.